src/Simplex_tree/include/gudhi/Simplex_tree.h

```diff
@@ -1497,6 +1497,56 @@ class Simplex_tree {
     return modified;
   }
 
+ public:
+  /** \brief Prune above dimension value given as parameter.
```

Comment:
Since that's already in the name of the function, I'd take the chance to paraphrase. Maybe "remove all simplices of dimension > `dim`" or something.

Reply:
I rolled back this bad find/replace on [7592ece](https://github.com/GUDHI/gudhi-devel/pull/837/commits/7592ece0cfd599359a9d3dd9375c316ef9440457)